Neos.Flow/Configuration/Objects.yaml

```diff
@@ -292,6 +292,9 @@ Neos\Flow\Property\PropertyMapper:
 # ResourceManagement                                                       #
 #                                                                          #
 
+Neos\Flow\Mvc\Routing\RouterInterface:
+  className: Neos\Flow\Mvc\Routing\Router
+
```

Comment:
You changed this in the section "ResourceManagement", There is already one for "MVC"